cnf-tests/.konflux/README.md

```diff
@@ -0,0 +1,40 @@
+**Update RPMs dependencies**
```

Comment:
@ralphbean  , Perhaps this README can be a good addition to the konflux docs as well ?